src/infrastructure/transaction/CreateTransactionFromDTO.ts

```diff
@@ -118,17 +118,27 @@ export const CreateTransactionFromDTO = (transactionDTO): Transaction => {
  */
 const CreateStandaloneTransactionFromDTO = (transactionDTO, transactionInfo): Transaction => {
     if (transactionDTO.type === TransactionType.TRANSFER) {
+        /**
```

Comment:
Not 100% sure if this is the best approach. Basically, if user uses serialisation method `toJson()' it will create a `Message` object rather than encoded message (from the rest endpoint for example). Open for suggestions

Reply:
As proposed on slack, this could be solved in a `extractMessage` auxiliary class.